.github/workflows/sonar-code-coverage.yml

```diff
@@ -0,0 +1,26 @@
+name: Dokazovi build status
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
I doubt that we need this info on the PR action, let's discuss.

Reply:
deleted until discussion